runtime/slice.go

```diff
@@ -156,6 +156,58 @@ func initSliceType(map[string]*Object) {
 	SliceType.slots.Repr = &unaryOpSlot{sliceRepr}
 }
 
+func sliceIndex(f *Frame, o *Object) (i int64, raised *BaseException) {
```

Comment:
return -1 from that function is not possible because it get later got adjusted, is it ok to return maxIntBig instead?